File: src/main/java/net/atos/zac/app/productaanvragen/model/RESTInboxProductaanvraagListParameters.java
/*
 * SPDX-FileCopyrightText: 2023 Atos
 * SPDX-License-Identifier: EUPL-1.2+
 */

package net.atos.zac.app.productaanvragen.model;

import net.atos.zac.app.shared.RESTListParameters;
import nl.info.zac.app.search.model.RestDatumRange;

public class RESTInboxProductaanvraagListParameters extends RESTListParameters {

    public String type;

    public RestDatumRange ontvangstdatum;

    public String initiatorID;
}

File: src/main/kotlin/nl/info/zac/app/productaanvraag/model/RestInboxProductaanvraagListParameters.kt
/*
 * SPDX-FileCopyrightText: 2023 Atos, 2026 INFO.nl
 * SPDX-License-Identifier: EUPL-1.2+
 */
package nl.info.zac.app.productaanvraag.model

import net.atos.zac.app.shared.RESTListParameters
import nl.info.zac.app.search.model.RestDatumRange
import nl.info.zac.util.AllOpen
import nl.info.zac.util.NoArgConstructor

@AllOpen
@NoArgConstructor
class RestInboxProductaanvraagListParameters : RESTListParameters() {
    var type: String? = null
    var ontvangstdatum: RestDatumRange? = null
    var initiatorID: String? = null
}
